extension/src/testConfigManager.ts

```diff
@@ -11,7 +12,9 @@ import { TestConfig } from './Models/testConfig';
 import * as Logger from './Utils/Logger/logger';
 
 export class TestConfigManager {
-    constructor(private readonly _configPath: string, private readonly _projectManager: ProjectManager) {
+    private readonly _configPath: string;
+    constructor(storagePath: string, private readonly _projectManager: ProjectManager) {
+        this._configPath = path.join(storagePath, 'configs', Configs.TEST_LAUNCH_CONFIG_NAME);

```

Comment:
If make this config file locate at the project specific user folder, how can user make changes to this file?

Reply:
we provide command `Java: Edit Test Configuration` for user to open config, but this is a good suggestion, I would move the config to `.vscode` folder where user could directly access it.